### PR TITLE
Feature/s6b 89

### PR DESCRIPTION
**Pull request by cushingbf**
Menu system, text note service, View Notes page, and Save Note page.

### CODE REVIEW

**Review by abaileys — COMMENTED** [review]
Ben,
I tried to save a note, Harkify said it saved correctly, but I got the error message that there were no notes to display.  I think I may need to create a file in the correct location, will look up where it needs to be and try it early tomorrow.  It's probably something on my end.

Alec

**Review by ghost — APPROVED** [review]
LGTM

**Comment by cushingbf** [comment]
Alec,
Were you using Chrome or a web-based emulator? That might be why you saw no messages. The back-end code I'm using can't seem to figure out where to save files if you use a web-based emulator. I noticed that as well. The android studio emulator worked fine though.

File: harkify/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-chrome",
```
Comment (willywil548) [inline harkify/.vscode/launch.json:8]: Did you mean to upload VS Code launch settings?

**Review by willywil548 — APPROVED** [review]

File: harkify/.vscode/launch.json (hunk 1 above)
Comment (cushingbf) [reply harkify/.vscode/launch.json:8]: Probably not. I was having problems even running the app after I merged in the main branch changes and I think this was one of my attempts to get it working. Do the launch settings have a significant effect? They didn't stop me from using the Android Studio emulator.

**Pull request merged**